src/index.ts

```diff
@@ -2217,7 +2215,7 @@ async function getXChainOptionsForLink({
 	})
 
 	const destinationChains = supportedChains
-		.filter((chain) => chain.chainId !== sourceChainId && chain.chainType === 'evm')
+		.filter((chain) => chain.chainType === 'evm')
```

Comment:
Question: idg this change

Reply:
this enables same chain swapping, not filtering out the source chain anymore

Reply:
ah got it!